x-pack/solutions/observability/plugins/observability_ai_assistant_app/public/rule_connector/ai_assistant.tsx

```diff
@@ -36,7 +36,11 @@ export function getConnectorType(
       actionParams: ObsAIAssistantActionParams
     ): Promise<GenericValidationResult<ObsAIAssistantActionParams>> => {
       const validationResult = {
-        errors: { connector: new Array<string>(), message: new Array<string>() },
+        errors: {
+          connector: new Array<string>(),
+          message: new Array<string>(),
+          prompts: new Array<string>(),
```

Comment:
Why not use array literals?

```suggestion
          connector: [] as string[],
          message: [] as string[],
          prompts: [] as string[],
```